### ISSUE TITLE
Web page refinements

### ISSUE THREAD

**Issue by baskaufs** [HeardLibrary/twine-for-wikicite#62]
Things that are wrong:
1. The video does not center when the screen width gets wider than the fixed video width.
2. The video does not shrink when the screen width gets narrower than the fixed video width.

**Comment by marjans74** [comment]
Button to select next video should appear after text of learning objectives for current video

**Comment by marjans74** [comment]
- [ ] Button to select next video should appear after text of learning objectives for current video

**Comment by CliffordAnderson** [comment]
- [ ] Buttons should be highligted on mouse over
- [ ] Button text should be centered

**Comment by baskaufs** [comment]
- [ ] There is too much dead space between the page title and the embedded video. On a laptop screen, it requires the user to scroll in order to see the video. Without the dead space the user could just hit the play button

**Comment by baskaufs** [comment]
@CliffordAnderson @awesolek2 @marjans74 @Talinum @cylew  I've added the developer's styles directly to the Twine and all seems to be working fine.

1. The fix for responsiveness of the iFrame involved an interesting hack I didn't know about. He used "! important" to force an override rather than actually figuring out what the problem was and fixing it. Not a very good programming practice, but a really quick and easy fix for the problem. If I'd known about that, we could have fixed the problem ourselves weeks ago.
2. It was pretty easy to use "inspect page" to figure out why that big blob of dead space was above the iFrame. I think originally it was there to let people see more of the background image, but getting rid of it makes viewing the videos much more sensible.
3. I looked at the web page on my iPhone and it's OK now. Not great but OK. The key thing is that when I click on the video, it goes to full screen and is therefore easy to watch. Closing out the full screen video then goes back to the web page where navigation can continue. I'm not sure if the full screen feature is enabled because I have the YouTube app installed on my phone or if it's a native feature of the iFrame. We definitely should get some people to look at this on different phones to understand how it behaves. I'll ask Jessie to look at it on her Android phone and tell me how it behaves.

So although there are probably some tweaks that would improve the page more, I'm satisfied that it is "good enough" for now. We may change some more things after user experience testing, but I'm going to close this issue out.
